pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -78,6 +78,7 @@ public static class Helix {
     public static final String IS_SHUTDOWN_IN_PROGRESS = "shutdownInProgress";
     public static final String QUERIES_DISABLED = "queriesDisabled";
     public static final String QUERY_RATE_LIMIT_DISABLED = "queryRateLimitDisabled";
+    public static final String DATABASE_QUERY_RATE_LIMIT = "databaseMaxQueriesPerSecond";
```

Comment:
nit: DATABASE_MAX_QUERIES... to be consistent with how other variables are named.